packages/backend/src/telemetry/service.ts

```diff
@@ -67,17 +75,16 @@ class TelemetryServiceImpl implements TelemetryService {
       resource: new Resource({ 'service.name': SERVICE_NAME })
     })
 
-    deps.collectorUrls.forEach((url) => {
-      const metricExporter = new OTLPMetricExporter({
-        url: url
-      })
-
-      const metricReader = new PeriodicExportingMetricReader({
-        exporter: metricExporter,
-        exportIntervalMillis: deps.exportIntervalMillis ?? 15000
+    this.meterProvider = new MeterProvider({
```

Comment:
we can delete the setting of `this.meterProvider` on L74 now

Reply:
whoops, yes. good catch